app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```

Comment:
Completely for curiosity reasons, but does the `<` indicate inheritance from a superclass?

Reply:
Yesssss spot on! The Application Controller class inherits from ActionController::Base, which is like the foundation for every controllers in Rails.